zetaclient/tss/healthcheck.go

```diff
@@ -89,16 +96,30 @@ func HealthcheckWorker(ctx context.Context, server *tss.TssServer, p Healthcheck
 		return nil
 	}
 
-	peersCounter := func(_ context.Context, _ *ticker.Ticker) error {
+	knownPeersCounter := func(_ context.Context, _ *ticker.Ticker) error {
 		peers := server.GetKnownPeers()
 		p.NumConnectedPeersMetric.Set(float64(len(peers)))
-		p.Telemetry.SetConnectedPeers(peers)
+		p.Telemetry.SetKnownPeers(peers)
```

Comment:
The line above should be set _known_ peers metrics

Reply:
Right I got confused during the rebase. I don't think we actually need knownpeers as a metric. I'll remove it.